neptune/internal/storage/datastream.py

```diff
@@ -0,0 +1,93 @@
+#
+# Copyright (c) 2019, Neptune Labs Sp. z o.o.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import hashlib
+import io
+import os
+import stat
+import tarfile
+
+from future.builtins import object
+
+
+class FileChunkWithCallback(object):
```

Comment:
Nie ma na to jakiejś bardziej sensownej nazwy, tj. od tego, jaką rolę pełni ta klasa?

Reply:
Może po prostu `FileChunk`